.github/workflows/ci.yml

```diff
@@ -13,8 +13,6 @@ on:
       - "*"
 
   pull_request:
-    branches:
-      - "*"
```

Comment:
i think we want this so we have CI run when making PR's to other branches?

Reply:
I had to remove this to get CI to run on my PR